api.planx.uk/modules/webhooks/service/updateTemplatedFlowEdits/helpers.ts

```diff
@@ -19,11 +23,13 @@ export const transformDeltaToTemplatedFlowEditsData = (
       updatedNodeData[updatedKey] = data[nodeId]?.["data"]?.[updatedKey];
     });
 
-    const updatedEdges = nodeData?.edges;
+    // Always preserve a templated folders' "edges", even if it's been "emptied" out
+    const updatedEdges =
+      nodeData?.edges || nodeData?.type === ComponentType.InternalPortal;
 
     templatedFlowEditsData[nodeId] = {
       ...(Object.keys(updatedNodeData).length > 0 && { data: updatedNodeData }),
-      ...(updatedEdges && { edges: data[nodeId]?.["edges"] }),
+      ...(updatedEdges && { edges: data[nodeId]?.["edges"] || [] }),
```

Comment:
Previous behavior would have been to omit the `edges` property all together if the node was edited to remove it's edges, but we want to explicitly store the empty array `[]` for correct reconciliation later